lib/matplotlib/backends/_backend_tk.py

```diff
@@ -790,9 +791,6 @@ def hidetip(self):
 
 
 class RubberbandTk(backend_tools.RubberbandBase):
-    def __init__(self, *args, **kwargs):
-        backend_tools.RubberbandBase.__init__(self, *args, **kwargs)
```

Comment:
It seems that `RubberbandBase` does not have an `__init__`. So technically, this change is possible.

Generally, is it wise not to call `super().__init__()`. The parent class may later add an `__init__()` method.

Here in particular, while `RubberbandBase` does not have `__init__()`, its parent `ToolBase` has, which does not get called also with the existing code. That doesn't look right to me.

Reply:
It will get called even after deleting this code block: when instantiating a RubberbandTk, Python will call `RubberbandTk.__init__`; because RubberbandTk does not define an `__init__`, it is inherited by looking up in the base classes; ultimately this resolves to ToolBase.__init__`.
In other words, the deleted code block does not do anything.